changed UI of blood availability page

crveda opened the pull request.

crveda commented:
Before:
![2022-10-05 (2)](https://user-images.githubusercontent.com/84971383/194053248-b7279757-718f-4097-a714-08968468d981.png)

After:
![2022-10-05 (1)](https://user-images.githubusercontent.com/84971383/194053488-b4e91c60-65c1-4cf3-a78c-17ef1fa4dc72.png)
![2022-10-05](https://user-images.githubusercontent.com/84971383/194053509-877a12de-3acf-469f-95ae-a47ff0256735.png)

crveda commented:
And can you please add Hacktoberfest label ?

crveda commented:
@deepanshu-sharmaa thank you sir.

deepanshu-sharmaa commented:
Hey @crveda can you do me a favour by changing the background of the page you modified to light colour or maybe some image with light colours?

crveda commented:
Yeah sure sir. Will start working on it.

crveda commented:
Sir I had updated the background color .
Can you please verify once before I make the PR ?

Updated :
![2022-10-06 (2)](https://user-images.githubusercontent.com/84971383/194235545-d68ac118-b22f-42a0-ba48-67b5a6ab5e77.png)
![2022-10-06](https://user-images.githubusercontent.com/84971383/194235567-c4ebea41-fe52-44f5-aab8-de04cdcab89b.png)

deepanshu-sharmaa commented:
@crveda yeah now it's looking decent. Thanks a lot.

crveda commented:
Okay sir . Thank you
